src/WORKSPACE

```diff
@@ -99,9 +99,9 @@ protobuf_deps()
 
 git_repository(
     name = "com_github_nelhage_rules_boost",
-    commit = "9f9fb8b2f0213989247c9d5c0e814a8451d18d7f",
+    commit = "4ab574f9a84b42b1809978114a4664184716f4bf",
     remote = "https://github.com/nelhage/rules_boost",
-    shallow_since = "1570056263 -0700",
+    shallow_since = "1694566326 +0000",
```

Comment:
@AdenC123 Can you updated your rules_boost to this commit. This should have the configs you need for linux_aarch64.